content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+
+<div class="contain">
+
+{{ embed.img(
+  src='blog/2023/yes.png',
+  alt='The word yes with a smiley and sparkles',
+  attrs={
+    'class': 'align-left'
+  }
+) }}
+
+## Choose an RWA when…
+### Lower Cost
+*…when it’s important to keep initial development costs low.*
+
+Responsive web apps use a single “codebase.” The term “codebase” is developer speak for the collection of code used to build an application. Compared to native mobile apps, RWAs are relatively inexpensive to develop, requiring just one codebase to be available on any device with a web browser.
+
+To be clear, building any kind of app is not a low cost endeavor. Even a very simple responsive web app can have a starting price of $10K, and very complex web apps, such as social media platforms or project management software, cost many millions of dollars to design, develop, and maintain.
+
+</div>
+
+### Faster Development
+*…when it’s a priority to make your app available to users quickly.*
+
+RWAs take less time to build, relative to native mobile apps; it only takes one codebase to reach any device with a browser. It can also take less time to make the minimum viable product (MVP) available to an initial set of users, get feedback, and release frequent, incremental updates.
+
+### Less Maintenance
+*…when you need to keep ongoing maintenance costs low.*
+
+RWAs require less time and expense to maintain compared to native mobile apps. An RWA’s single codebase is built with standard web technologies, and doesn’t require an update of the entire app to make incremental improvements. It also takes less time and fewer developers to update a single codebase than to update separate codebases that use specialized technology for multiple platforms.
+
+### Discoverable Via Search Engine
+*…when the people you want to reach will look for your content with a search engine.*
+
+Search engines, like Google, “crawl” the web looking for keywords and other information to sort and rank the content. Then they collect, process, and store a database of all that content. This is called “indexing.” RWAs live on the web, so unless special code is added to prevent it, they will automatically be indexed and appear in organic search results.
+
+### Broad Reach
+*…when you want to reach a broad audience across devices.*
+
+People can access the same RWA on any device from a small iPhone to an Android tablet, from a MacBook laptop to a giant Windows desktop, unlike native mobile apps which are limited to supported mobile devices. Additionally, the web provides a high level of “backward compatibility,” meaning RWAs can be available in some form even on old devices. On the other hand, one platform update can totally block a device from accessing a native mobile app.
```

Comment:
MacBook laptop is like Sahara Dessert. In this case, since a MacBook is always a laptop, we can probably drop either MacBook or laptop.

Reply:
Also, do we need quotes around backwards compatibility? Literally asking since I don't know what's technically correct versus just what I feel in my bones :)